### PR TITLE
Submitting completed project

### PR DESCRIPTION
**Pull request by saypop**

### CODE REVIEW

**Comment by JRRS1982** [comment]
All tests pass, includes feature and unit tests, has 100% coverage and excellent breakdown of the steps taken to complete the challenge within the readme. Edge cases tested, commit messages clear and code is pretty easy to understand. Digging deeper now.

File: spec/airport_spec.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require 'airport'
+describe Airport do
+
+  # test described in Step 04 in the README
```
Comment (JRRS1982) [inline spec/airport_spec.rb]: Not sure if comments like this are good or bad, something to look into. Seems like a lot of work!

File: README.md [hunk 2]
```diff
@@ -69,11 +69,10 @@ When submitted this program had:
 24. Refactoring: removed redundant test in for airport class, rewrote tests for weather, used private method to check if plane is able to proceed.
 25. Planes should not land in stormy weather so the test should show that a plane is flying after it tries to land in stormy weather.
 26. Time for a commit: "Planes can now only land in good weather."
-27. Refactoring: none required.
```
Comment (JRRS1982) [inline README.md]: I really like how you have kept up to date with your readme as you coded, but seems like a lot of work.
